crates/cxx-qt-gen/src/generator/rust/mod.rs

```diff
@@ -146,6 +146,48 @@ pub fn get_params_tokens(
     }
 }
 
+/// Return the [TokenStream] of the parsed parameters, with self type qualified for use in generation
+pub fn get_params_tokens_qualified(
+    mutable: bool,
+    parameters: &[ParsedFunctionParameter],
+    class_name_path: &Path,
+) -> TokenStream {
+    let struct_sig = if mutable {
+        quote! { Pin<&mut #class_name_path> }
+    } else {
+        quote! { &#class_name_path }
+    };
+    if parameters.is_empty() {
+        quote! { self: #struct_sig }
+    } else {
+        let parameters = parameters
+            .iter()
+            .map(|parameter| {
+                let ident = &parameter.ident;
+                let ty = &parameter.ty;
```

Comment:
i think you need to run `syn_type_cxx_bridge_to_qualified` over this? Otherwise if the type is `T` from the bridge it won't be converted to `ffi:T` or whatever etc